store/sqlstore/upgrade.go

```diff
@@ -20,6 +20,7 @@ import (
 )
 
 const (
+	CURRENT_SCHEMA_VERSION   = VERSION_5_15_0
```

Comment:
why is 5.15 and not 5.16?

Reply:
This models the current behaviour. The db schema version stays at `5.15` until https://github.com/mattermost/mattermost-server/blob/6e4a8522ff13a015160f4f92133894e7d8679ebe/store/sqlstore/upgrade.go#L748 is commented out, which we will do shortly after the release branch is cut.

Reply:
ok